.gitignore

```diff
@@ -1,3 +1,2 @@
 node_modules
-dist
```

Comment:
Should `/dist` and `package-lock.json` stay in .gitignore?

Reply:
`Dist `must stay at .gitignore because this is the code that I send to npm. It is not necessary at git hub.
This project was configured with yarn, because of this, yarn.lock is in .gitignore.